### PR TITLE
py3: enable py3 pylint

### PR DESCRIPTION
**Pull request by mbasti-rh**
We should run pylint in both python2 and python3 versions

### CODE REVIEW

File: .test_runner_config.yaml [hunk 1]
```diff
@@ -40,7 +40,8 @@ steps:
     -a ${server_password} --setup-dns --auto-forwarders
   - ipa-kra-install -p ${server_password}
   lint:
-  - make V=0 lint
+  - PYTHON=/usr/bin/python2 make V=0 lint
+  - PYTHON=/usr/bin/python3 make V=0 lint
```
Comment (tiran) [inline .test_runner_config.yaml]: please make the python3 target ```pylint``` only. I just realized that lint also runs the i18n and JS linters.

Comment (mbasti-rh) [inline .test_runner_config.yaml]: I don't think that running those linters twice is an issue, but as you wish I updated PR

**Comment by tiran** [comment]
It's just a minor performance improvements to speed up CI.

**Comment by mbasti-rh** [comment]
Lint part of travis-CI passed

**Comment by mbasti-rh** [comment]
Fixed upstream
master:
https://fedorahosted.org/freeipa/changeset/d648c6a6925298a1db0c61381d72b6c4d0500c10

**Pull request closed without merging**